packages/react-instantsearch-hooks-dom/src/ui/ShowMoreButton.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+type ShowMoreButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> & {
+  isShowingMore: boolean;
+};
+
+export function ShowMoreButton({
+  isShowingMore,
+  ...props
+}: ShowMoreButtonProps) {
+  return (
+    <button {...props}>{isShowingMore ? 'Show less' : 'Show more'}</button>
```

Comment:
How could we add localization for this button? In RIS, for example, there's a [`translations`](https://www.algolia.com/doc/api-reference/widgets/refinement-list/react/#widget-param-translations) prop that can help with this.

Reply:
We discussed it in another PR with @Haroenv and there's [this RFC](https://github.com/algolia/instantsearch-rfcs/blob/5414147664e12457136f50bb9511c9081705929f/accepted/translations.md) which we could adopt.

Reply:
I was thinking of something "simpler" in the meantime, like the `resetLabel` prop we have for `<ClearRefinements>` for example. It's fine for me if that's in another PR.

Reply:
It's not only this prop, we have five of them.

What we can do is leave this PR as is for now (we did merge some other widgets without translations) and add a new task to introduce the Translations API, then implement it in all widgets. WDYT?

Reply:
Works for me!